lib/globalize/active_record/adapter_dirty.rb

```diff
@@ -0,0 +1,64 @@
+require 'pry'
```

Comment:
@ASnow sorry, I know this is from a while ago.. but can this `require` be removed please?